services/services.rst

```diff
@@ -0,0 +1,209 @@
+Overview of services
+******
+
+Below are tools and resources that are used in the PFS project. 
+Many services are hosted at IPMU under `pfs.ipmu.jp <https://pfs.ipmu.jp>`_ 
+(mainly for instrument hardware development) or 
+`pfspipe.ipmu.jp <pfspipe.ipmu.jp>`_ (for software development), and also PFS 
+project is using several external services, like 
+`GitHub Subaru-PFS organization <https://github.com/Subaru-PFS>`_. 
+You may need account(s) and access to some or all of them depending on your 
+roles and contributions to the project, most of services hosted at IPMU are 
+under single sign on (SSO) by pfs.ipmu.jp. 
+Refer `account policy <account.rst>`_ for details and registration. 
+
+* `SuMIRe/PFS PBworks wiki`_
+* `Public Web Site at pfs.ipmu.jp`_
+* `PFS project internal website (pfs.ipmu.jp)`_
```

Comment:
the same as above:
pfs.ipmu.jp (https) ?

Reply:
added http/https